src/ktn_dodger.erl

```diff
@@ -74,6 +73,8 @@
 
 %% We have snake_case macros here
 -elvis([{elvis_style, macro_names, disable}]).
+-elvis([{elvis_style, no_catch_expressions, disable}]).
+-elvis([{elvis_style, no_throw, disable}]).
```

Comment:
I could've "tried" to fix this, but I didn't want to deal with the consequences of that action 😆 so I avoid it.

Reply:
Fine with me.